chromium_src/chrome/browser/ui/views/frame/browser_view_layout.h

```diff
@@ -15,9 +15,14 @@
 #define LayoutBookmarkAndInfoBars virtual LayoutBookmarkAndInfoBars
 #define LayoutInfoBar virtual LayoutInfoBar
 #define LayoutContentsContainerView virtual LayoutContentsContainerView
+#define set_contents_border_widget           \
```

Comment:
maybe we should add a comment about the purpose of these overrides.

Reply:
I think this comment can be improved a bit. Right now it is describing something I already can see just by looking at the override itself. It would better if we could describe "why" we are adding these methods in particular.

Reply:
Good call. Updated.